.gitignore

```diff
@@ -1,5 +1,6 @@
 *.a
 *.bak
+*.cmake
```

Comment:
It is convention for CMake sources to either be named CMakeLists.txt or *.cmake.

So we would normally not want to gitignore *.cmake.

Reply:
Thanks for the comment. I wanted to target files autogenerated by CMake. I have pushed updates to be more specific. Thoughts?

Reply:
Great, those files are safe to ignore.